queue_services/account-mailer/q_cli.py

```diff
@@ -220,56 +106,23 @@ def subscription_options():
-                                    "accountnumber=", "ordernumber=", "amount=", "transactionid=", "transactiontime=",'cfsaccountid='])
-    except getopt.GetoptError:
-        print('q_cli.py -o <old_identifier> -n <new_identifier>')
+        opts, args = getopt.getopt(sys.argv[1:], "hp:r", ["payload=", "replace="])
+    except getopt.GetoptError as e:
+        print(e)
         sys.exit(2)
 
-    auth_account_name = bank_number = bank_branch_number = \
-        bank_account_number = order_number = transaction_amount = transaction_id = auth_username = transaction_time = cfs_account_id = None
+    replace_payload: bool = False
 
     for opt, arg in opts:
         if opt == '-h':
-            print('q_cli.py -o <old_identifier> -n <new_identifier>')
             sys.exit()
-        elif opt in ("-m", "--mode"):
-            mode = arg  # pad confirmation - "pad", refund request - "refund", acc suspended - "acc_suspend", acc restored - "acc_restore"
-        elif opt in ("-i", "--id"):
-            auth_account_id = arg
-        elif opt in ("-n", "--name"):
-            auth_account_name = arg
-        elif opt in ("-u", "--username"):
-            auth_username = arg
-        elif opt in ("-b", "--banknumber"):
-            bank_number = arg
-        elif opt in ("-t", "--transitnumber"):
-            bank_branch_number = arg
-        elif opt in ("-a", "--accountnumber"):
-            bank_account_number = arg
-        elif opt in ("-o", "--ordernumber"):
-            order_number = arg
-        elif opt in ("-p", "--amount"):
-            transaction_amount = arg
-        elif opt in ("-d", "--transactionid"):
-            transaction_id = arg
-        elif opt in ("-z", "--transactiontime"):
-            transaction_time = arg
-        elif opt in ("-c", "--cfsaccountid"):
-            cfs_account_id = arg
+        elif opt in ("-p", "--payload"):
+            payload = json.loads(arg)
+        elif opt in ("-r", "--id"):
+            replace_payload = str(arg).lower() == 'true'
 
     event_loop = asyncio.get_event_loop()
-    event_loop.run_until_complete(
-
-        run(event_loop, mode, auth_account_id, auth_account_name, auth_username, bank_number,
-            bank_branch_number, bank_account_number, order_number, transaction_amount, transaction_id,
-            transaction_time, cfs_account_id))
+    event_loop.run_until_complete(run(event_loop, payload, replace_payload))
 
-# pad cmd --> python3 q_cli.py -m pad -i 10 -n TestAccount -b 088 -t 00277 -a 12874890
-# refund cmd --> python3 q_cli.py -m refund -i 10 -o 67892 -p 25.33 -d 988
-# account suspended cmd --> python3 q_cli.py -m acc_lock -i 4 -n SomeAccount
-# account restored cmd --> python3 q_cli.py -m acc_unlock -i 4 -n SomeAccount
-# account conf pver-- >  python3 q_cli.py -m conf_per -i 4
-# account conf pver-- >  python3 q_cli.py -m pad_invoice -i 4
-# account admin removal-- >  python3 q_cli.py -m admin_removed -i 4
+# Examples:
```

Comment:
do you mind adding one more sample here? 
mostly many non-python devs use this to copy the command.. just one sample will be helpful

Reply:
Done